src/ExtensionsSample/Samples/DocumentDBSamples.cs

```diff
@@ -27,13 +27,19 @@ public static class DocumentDBSamples
         //   IAsyncCollector<T>
         //   ICollector<T>
         public static void InsertDocument(
-            [TimerTrigger("00:01")] TimerInfo timer,
-            [DocumentDB("ItemDb", "ItemCollection")] out ItemDoc newItem)
+            [TimerTrigger("00:01", RunOnStartup = true)] TimerInfo timer,
+            [DocumentDB("ItemDb", "ItemCollection")] out ItemDoc newItem,
```

Comment:
If someone wants to output two items to the same collection, it would be better for them to us the collector binding, right? Why are you showing this pattern?